IOMC/EventVertexGenerators/src/BeamProfile2DB.cc

```diff
@@ -48,7 +48,6 @@ class BeamProfile2DB : public edm::global::EDAnalyzer<> {
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
 
 private:
-  void beginJob() override;
```

Comment:
This file looks better placed under `CondTools/BeamSpot`.

Reply:
addressed in [2bc4a06](https://github.com/cms-sw/cmssw/pull/41918/commits/2bc4a066bdf232e38ca496a0d6be5b75c354654b)